### ISSUE TITLE
Python script prints out to command line but doesn't send message over websocket using websocketd 

### ISSUE THREAD

**Issue by Alikerin** [joewalnes/websocketd#211]
After installing websocketd, I tested it with count.sh and it worked fine. I also opened the devconsole and I saw the messages.
On writing my own python script which is meant to count also, I found out that when I execute it in the command line it works fine. but after binding it to websocketd I could not see any message received.
I tried running websocketd with devconsole but still, nothing shows up. 

```
#!/usr/bin/python
import RPi.GPIO as G
import time

count = 0.0

G.setmode(G.BCM)
G.setup(23, G.IN)#connected to pin 7 on arduino where a 1 cyvle pulse train is implemented in hardware
G.setup(24, G.IN)


def counter(pin):
	 #Checks the value of pin24 for increment or reduction
        global count
	state = G.input(24)#stores the state of GPIO24
        if state == False: #implies the robot is moving forward
		count = count + 0.013 #increment the counter
        elif state == True: #implies robot is moving backward
		count = count - 0.013 #reduce the counter

G.add_event_detect(23, G.RISING)#event added to GPIO23 for detecting a rising or falling edge
G.add_event_callback(23, counter)                        

while True:
	print(count)
	time.sleep(2)
```
			   
I will be glad it someone can help fix this issue.

**Comment by asergeyev** [comment]
Try ```stdout.flush()```  after print

**Comment by Alikerin** [comment]
Okay. I will try it and get back to you.

Thank you.

**Comment by Alikerin** [comment]
I have tried it and its still the same.

Thank you.

Below are some screenshots I have taken. 
![ws](https://cloud.githubusercontent.com/assets/15312963/15514273/59817c70-21b6-11e6-80d8-4a07a0f2f378.png)
![ws1](https://cloud.githubusercontent.com/assets/15312963/15514272/597ea266-21b6-11e6-9c3e-995999a1229c.png)
![ws2](https://cloud.githubusercontent.com/assets/15312963/15514274/59855bb0-21b6-11e6-9445-c3833d93bf11.png)

**Comment by asergeyev** [comment]
I wonder if script would need more stuff from environment than available...

I'd suggest you run it with higher log levels. And also try in your console that ```sudo env -i ./counter.py``` would work, that somewhat simulating clean-slate environment that websocket uses (although there are discrepancies...)

**Comment by Alikerin** [comment]
After going through http://www.perrygeo.com/raspberry-pi-real-time-sensor-plots-with-websocketd.html I realised that he had a python script which prints out some sensor values which he wants to make available to a websocket client.

I followed the steps, by first redirecting the output of my python script to a log.txt file, then using tail -f command to send it over the websocket.

But I included sys.stdout.flush() after the print statement, and everything worked out FINE.

Thank you.
